browser/ui/webui/brave_education/brave_education_handler.h

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+// based on //chrome/browser/ui/webui/whats_new/whats_new_handler.h
+
+#ifndef BRAVE_BROWSER_UI_WEBUI_BRAVE_EDUCATION_BRAVE_EDUCATION_HANDLER_H_
+#define BRAVE_BROWSER_UI_WEBUI_BRAVE_EDUCATION_BRAVE_EDUCATION_HANDLER_H_
+
+#include "brave/components/brave_education/education_urls.h"
+#include "brave/browser/ui/webui/brave_education/brave_education.mojom.h"
+// #include "components/user_education/webui/whats_new_registry.h"
```

Comment:
remove

Reply:
Fixed in https://github.com/brave/brave-core/pull/26154/commits/b8754b2a9bd54be5fe0c2216ea7684b5bd523bad